src/controllers/ConfirmationController.ts

```diff
@@ -0,0 +1,14 @@
+import {controller, httpGet, BaseHttpController} from 'inversify-express-utils';
+import {CONFIRMATION_PAGE_URI} from '../utils/Paths';
```

Comment:
`{ CONFIRMATION_PAGE_URI }` instead of `{CONFIRMATION_PAGE_URI}` looks better, same for other imports